apps/prairielearn/src/lib/question-render.sql

```diff
@@ -249,24 +249,14 @@ SELECT
       )
     )
   ) AS assessment_instance,
-  jsonb_set(
-    jsonb_set(
-      to_jsonb(iq.*),
-      '{assigned_grader_name}',
-      to_jsonb(COALESCE(agu.name, agu.uid))
-    ),
-    '{last_grader_name}',
-    to_jsonb(COALESCE(lgu.name, lgu.uid))
-  ) AS instance_question
```

Comment:
The values from this query were actually not used anywhere in the code that I could find. As a matter of fact, I'm not even sure if this `instance_question` is used at all, since the code that calls it actually uses `instance_question` from res.locals, however since the lack of typing in res.locals does not give me an easy way to determine if this is used I kept it for now.